lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -227,7 +227,7 @@ runGhcid
   => FilePath -- ^ Path to .ghci
   -> Maybe String -- ^ Optional command to run at every reload
   -> m ()
-runGhcid dotGhci mcmd = callCommand $ unwords $ "ghcid" : opts
+runGhcid dotGhci mcmd = callCommand $ unwords $ $(staticWhich "ghcid") : opts
```

Comment:
Do you know how this working at all without ghcid being provided to the shell? I don't remember ever needing to install ghcid in order to use `ob run`, both before and after we switched to a pure shell.

Reply:
I think reflex-platform provides ghcid for us, so you always got it unless you really messed with the nix project expressions.